tests/congruence_tests/test_common.py

```diff
@@ -118,7 +118,7 @@ def compare_scored_record(
         point1.id == point2.id
     ), f"point1[{idx}].id = {point1.id}, point2[{idx}].id = {point2.id}"
     assert (
-        point1.score - point2.score < 1e-4
+        abs(point1.score / point2.score - 1) < 1e-4
```

Comment:
Is it actually better?
I can see the concern about abs, but with the division it is not clear for me
I don't mind, but it should be checked for a division by zero

Reply:
new reco was failing because for only negatives recommendation, scores can be in the magnitude of hundreds, which needed to either 
1. Decrease precision check
or
2. "Normalize" the scores

But then it is true that we get into division by zero errors

Reply:
Resorted to another solution: adjust the precision based on the magnitude of the score